contrib/IECoreArnold/src/IECoreArnold/ShapeAlgo.cpp

```diff
@@ -176,6 +176,17 @@ void convertRadius( const std::vector<const IECore::Primitive *> &samples, AtNod
 void convertPrimitiveVariable( const IECore::Primitive *primitive, const PrimitiveVariable &primitiveVariable, AtNode *shape, const char *name )
 {
 
+	// make sure the primitive variable doesn't clash with built-ins
+	const AtNodeEntry *entry = AiNodeGetNodeEntry( shape );
+	if ( AiNodeEntryLookUpParameter(	entry, name ) != NULL ){
+		msg(
+			Msg::Warning,
+			"ToArnoldShapeConverter::convertPrimitiveVariable",
```

Comment:
Yeah, I copied that from somewhere else and thought it might be a convention I didn't know about yet ;) I will replace all references to `ToArnoldShapeConverter`.